pom.xml

```diff
@@ -612,6 +614,9 @@
           <showDeprecation>true</showDeprecation>
           <contextPath>/jenkins</contextPath>
           <!-- TODO specify ${java.level} when JENKINS-20679 implemented -->
+          <systemProperties>
+            <hudson.Main.development>true</hudson.Main.development>
```

Comment:
yes, but through an intermediate POM that still had the previous version of the changes, bah, whatever, stupid mistake, fixed. Thanks!